pkg/controller/apischeme/apischeme_controller.go

```diff
@@ -96,5 +100,300 @@ func (r *ReconcileAPIScheme) Reconcile(request reconcile.Request) (reconcile.Res
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
-	return reconcile.Result{}, err
+
+	ownerTags, err := utils.AWSOwnerTag(r.client)
```

Comment:
does this return the AWS tags that come by default from the product?

I want to make sure we are tagging our items, everything we create in AWS, the ELB, SG's, etc, with the same tags the product uses.  This way it can be cleaned up during an uninstall.

Reply:
it does, from what I see below.